src/test/test_helpers.ts

```diff
@@ -53,10 +53,14 @@ export async function fakeResults(configFile: ConfigFile):
     ];
     for (let measurementIndex = 0; measurementIndex < measurement.length;
          measurementIndex++) {
+      const resultName = measurement.length == 1 ?
```

Comment:
`===` (nit that is causing the lit error)

Reply:
Oh whoops, thanks!

Reply:
Fixed! Build is now failing cuz 1 e2e test on MacOS failed. I assume that's unrelated